src/examples/to-hash-input.ts

```diff
@@ -0,0 +1,179 @@
+import {
```

Comment:
this is a sort of unit test, which is also run in Mina CI. I left it here just because for the Mina version, I had to remove the TS types, and it's easier to read & modify with the types in place